tardis/adapters/sites/htcondor.py

```diff
@@ -116,13 +225,8 @@ async def deploy_resource(
             ),
         )
 
-        response = await self._executor.run_command(
-            "condor_submit", stdin_input=submit_jdl
-        )
-        pattern = re.compile(
-            r"^.*?(?P<Jobs>\d+).*?(?P<ClusterId>\d+).$", flags=re.MULTILINE
-        )
-        response = AttributeDict(pattern.search(response.stdout).groupdict())
+        job_id = await self._condor_submit(submit_jdl)
```

Comment:
I am just afraid that a removal will break backwards compatibilty. Officially, we should deprecate it first and remove it later on in `0.8.0`.  So for example checking it is in there and print a warning like we did in https://github.com/MatterMiners/tardis/blob/9d45bbcdd15ca99e3c24aa593ebc3e49a81eb28a/tardis/adapters/sites/slurm.py#L58-L61, otherwise add `queue 1`.

Reply:
I've added a helper that checks for ``queue`` statements and inserts one otherwise.

Reply:
Great! Thanks!